com.nickgirga.chocolate-box.yml

```diff
@@ -0,0 +1,38 @@
+id: com.nickgirga.chocolate-box
+runtime: org.gnome.Platform
+runtime-version: '45'
+sdk: org.gnome.Sdk
+command: chocolate-box
+
+base: org.winehq.Wine
+base-version: stable-23.08
+
+finish-args:
+  # X11 + XShm access
+  - --share=ipc
+  - --socket=fallback-x11
+  # Wayland access
+  - --socket=wayland
+  # access to org.freedesktop.Flatpak; for flatpak Source port launching
+  - --talk-name=org.freedesktop.Flatpak
+  # GPU acceleration if needed
+  - --device=dri
+  # access to home files; for locating WADs and custom installed source ports
+  - --filesystem=home
```

Comment:
what the point of this you punch the other hole to run things out of the sandbox?